ngraph/test/onnx/onnx_import_quant.in.cpp

```diff
@@ -307,27 +307,6 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_model_quant_conv_linear_3d)
     test_case.run();
 }
 
-NGRAPH_TEST(${BACKEND_NAME}, onnx_model_qlinear_matmul)
-{
-    auto function = onnx_import::import_onnx_model(
-        file_util::path_join(SERIALIZED_ZOO, "onnx/qlinear_matmul.prototxt"));
```

Comment:
When you remove a test that uses a model, you need to remove the model too since it becomes obsolete. Please review the removed tests (not only in this file) and remove the dangling .prototxt files.

Reply:
Done.